.github/workflows/ci.yml

```diff
@@ -21,33 +21,52 @@ jobs:
            arch: 'arm64'
            cross-prefix: ' '
            cflags: '-DFORCE_AARCH64'
-           nix-shell: ci
          - runner: pqcp-arm64
            name: 'ubuntu-latest (aarch64)'
            arch: 'aarch64'
            cross-prefix: ' '
            cflags: '-DFORCE_AARCH64'
-           nix-shell: 'ci'
          - runner: pqcp-x64
            name: 'ubuntu-latest (x86_64, cross)'
            arch: 'x86_64'
            cross-prefix: 'aarch64-unknown-linux-gnu-'
            cflags: '-DFORCE_AARCH64'
-           nix-shell: 'x86_64-linux-cross-ci'
          - runner: pqcp-x64
            name: 'ubuntu-latest (x86_64, native)'
            arch: 'x86_64'
            cross-prefix: ''
-           nix-shell: 'ci'
+         - runner: ubuntu-latest
+           name: 'ubuntu-latest (x86_64, cross)'
+           arch: 'x86_64'
+           cross-prefix: 'aarch64-unknown-linux-gnu-'
+         - runner: ubuntu-latest
+           name: 'ubuntu-latest (x86_64, native)'
+           arch: 'x86_64'
+           cross-prefix: ''
```

Comment:
@hanno-becker do you think this is nice to have ? Which allows forks to run ci for linux.